emails/views.py

```diff
@@ -135,16 +137,25 @@ def wrapped_email_test(request):
     else:
         has_attachment = True
 
-    if "has_email_tracker_study_link" in request.GET:
-        has_email_tracker_study_link = strtobool(
-            request.GET["has_email_tracker_study_link"]
+    if "has_tracker_report_link" in request.GET:
+        has_tracker_report_link = strtobool(request.GET["has_tracker_report_link"])
+    else:
+        has_tracker_report_link = False
+    if has_tracker_report_link:
+        tracker_report_link = "https://example.com/fake_tracker_report_link"
```

Comment:
nit: This could be a real report to test that as well:
```suggestion
        tracker_report_link = '/tracker-report/#{"sender": "sender@example.com", "received_at": 1658434657, "trackers": {"fake-tracker.example.com": 2}}'
```